lib/serviceProvider.js

```diff
@@ -130,18 +141,23 @@ function doSoapRequest(url, method, soapAction, data, inputModel, outputModel, n
     body: soapBody,
     timeout: coreSettings.timeout || 15000
   };
+  var debug = {
+    startTime: new Date(),
+    request: {
+      method: method,
+      input: data,
+      request: soapRequest
+    }
+  };
+  if (coreSettings.debugSoap) coreSettings.logger(debug);
```

Comment:
If no requests are making it back, we'll have a pretty obvious problem, if it's only some I don't think it'll be obvious from the log output which ones (all the successes log this too, so there'll be a steady baseline amount of output/noise from this line)